login.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="login.css">
+  <title>판다마켓</title>
+</head>
+<body>
+  <div class="wrap">
+  <header>
+    <div>
+      <a href="/"><img src="./src/logo_pandamarket.png"></a>
+    </div>
+  </header>
+  <main> 
+    <div class="login">
+      <form class="loginform">
+        <div class="input_container">
```

Comment:
```suggestion
        <div class="input-container">
```

naming convention은 여러가지를 혼용하는 것 보다, 하나로 통일해서 사용하는게 눈에도 잘 익고 코드를 보는 사람이 읽기 편합니다.

일반적으로 html은 kebab-case를 사용하곤 합니다.

법으로 정해진건 없고, 팀단위 개발할 때에 팀에서 컨벤션을 정하고 컨벤션대로 사용해도 무방합니다.